WordPress/Classes/ViewRelated/Blog/Site Picker/SitePickerViewController+SiteActions.swift

```diff
@@ -0,0 +1,101 @@
+import UIKit
+import SwiftUI
+
+extension SitePickerViewController {
+
+    func makeSiteActionsMenu() -> UIMenu? {
+        UIMenu(options: .displayInline, children: [
+            UIDeferredMenuElement.uncached { [weak self] in
+                $0(self?.makeSections() ?? [])
+            }
+        ])
+    }
+
+    private func makeSections() -> [UIMenu] {
+        [
+            makePrimarySection(),
+            makeSecondarySection(),
+            makeTertiarySection()
+        ]
+    }
+
+    private func makePrimarySection() -> UIMenu {
+        UIMenu(options: .displayInline, children: [
+            MenuItem.visitSite(visitSiteTapped),
+            MenuItem.switchSite(siteSwitcherTapped)
```

Comment:
Should it only be displayed if there is more than one site?

Reply:
No, I think we should always show it - to add a new site you need to access the My Sites screen
<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/7b3f1576-0c5c-4efb-be81-5f9267ea66e5" width=200>

Reply:
Yes I agree. The action should always be there in the menu. but the 'tap icon to switch site' idea - would only be for multiple sites I think. If we do that.

Reply:
Tapping "Switch Site" is not the most obvious way to add a new site. I'd suggest adding two new actions "Create Site" and "Share" to the new menu.

Reply:
Fair point. I'll look into this after the header gradient.